Source/RunActivity/Viewer3D/Scenery.cs

```diff
@@ -480,10 +480,16 @@ public WorldFile(Viewer viewer, int tileX, int tileZ, bool visible)
                     }
                     else if (worldObject.GetType() == typeof(StaticObj))
                     {
-                        //          preTestShape for lookup if it is an animated clock shape with subobjects named as clock hands 
+                        // preTestShape for lookup if it is an animated clock shape with subobjects named as clock hands 
                         StaticShape preTestShape = (new StaticShape(viewer, shapeFilePath, worldMatrix, shadowCaster ? ShapeFlags.ShadowCaster : ShapeFlags.None));
-                        var animNodes = preTestShape.SharedShape.Animations?[0]?.anim_nodes ?? new List<anim_node>();
```

Comment:
Wouldn't it be shorter to do simply
`var animNodes = preTestShape.SharedShape.Animations?.FirstOrDefault()?.anim_nodes ?? new List<anim_node>();`

Reply:
I agree, this is much simpler and fits what needs to be fixed.

Reply:
Thanks, Peter.
Your suggestion now implemented.